syntax = "proto3";

package contracts.services.gateway.operations;

import "contracts/services/operations/operation.proto";

message MakePurchaseOperationRequest {
  contracts.services.operations.OperationStatus status = 2;
  float amount = 3;
  string card_id = 4;
  string category = 5;
  string account_id = 7;
}

message MakePurchaseOperationResponse {
  contracts.services.operations.Operation operation = 1;
}